lib/src/tool_bar.dart

```diff
@@ -793,6 +863,12 @@ class ToolBarState extends State<ToolBar> {
         return {'format': 'header', 'value': isActive ? 1 : 4};
       case ToolBarStyle.headerTwo:
         return {'format': 'header', 'value': isActive ? 2 : 4};
+      case ToolBarStyle.headerThree:
+        return {'format': 'header', 'value': isActive ? 3 : 4};
```

Comment:
Whenever we deselect any header value  its selects the h4  by default 
so value should be false when any header is not selected 
So Please change value is notActive as false instead of 4